docs/ops/activation/SoftPlus_4.md

```diff
@@ -19,16 +19,19 @@ SoftPlus(x) = \left\{\begin{array}{r}
 \end{array}\right.
 \f]
 
-**Note**: For numerical stability the operation reverts to the linear function when `x > threshold` where `threshold` depends on *T* and
-is chosen in such a way that the difference between the linear function and exact calculation is no more than `1e-6`.
-The `threshold` can be calculated with the following formula where `alpha` is the number of digits after the decimal point,
-`beta` is maximum value of *T* data type:
+**Note**: 
```

Comment:
@lazarevevgeny Could you please take a look?

Reply:
I don't agree with these changes. @a-sidorova did analysis and provided a formulae to calculate the threshold based on the data type. We should use this approach and do not hardcode any constants in the specification.

Reply:
ok, revert back. update after getting alignemnt by email.